cypress/e2e/about.cy.ts

```diff
@@ -2,9 +2,17 @@ import enUS from '../../i18n/locales/en.json'
 
 describe('About page', () => {
     context('Desktop resolution', () => {
+        before(() => {
```

Comment:
do we need before AND beforeeach? seems weird to have both 🤔

Reply:
@ermish So for this specific context, we only need to visit `/about` once and all of the tests can run after it loads. It takes quite a while for it to fully render—probably because of the images and whatnot. I figured it was probably best to load the page once, wait for it to finish loading, and then use the `beforeEach()` just to make sure the resolution stays at 1920x1080 for each test. Open to suggestions on how to better implement this though!

Reply:
we can improve it in future commits 👍  I have some ideas. For now, let's get this in to fix the tests